LoopKitUI/Views/QuantityPicker.swift

```diff
@@ -34,10 +34,17 @@ public struct QuantityPicker: View {
         stride: HKQuantity,
         guardrail: Guardrail<HKQuantity>,
         formatter: NumberFormatter? = nil,
-        isUnitLabelVisible: Bool = true
+        isUnitLabelVisible: Bool = true,
+        guardrailColors: GuardrailColors = GuardrailColors()
```

Comment:
Why not pull from the environment in QuantityPicker as well?

Reply:
^ good question.

Reply:
the environment property isn't available in the init.

Reply:
the environment property isn't available in init